LINK/etc/default/evm

```diff
@@ -25,6 +25,9 @@ export QPID_SSL_CERT_DB=/etc/qpid/certs
 # Location of certificates and provider keys
 export KEY_ROOT=/var/www/miq/vmdb/certs
 
+export APPLIANCE_REPO_DIRECTORY=/opt/manageiq/manageiq-appliance
```

Comment:
When it's an rpm it's not a repo, which is what I thought was confusing.  But yeah, I agree with you on "ROOT".